doc/lsst.ctrl.bps/quickstart.rst

```diff
@@ -367,9 +367,11 @@ Supported settings
 
        pipetask:
          pipetask_init:
-           runQuantumCommand: "${CTRL_MPEXEC_DIR}/bin/pipetask --long-log run -b {butlerConfig} -i {inCollection} --output-run {outCollection} --init-only --skip-existing --register-dataset-types --qgraph {qgraphFile} --no-versions"
+           runQuantumCommand: "${CTRL_MPEXEC_DIR}/bin/pipetask --long-log run -b {butlerConfig} -i {inCollection} --output {output} --output-run {outCollection} --init-only --skip-existing --register-dataset-types --qgraph {qgraphFile} --no-versions"
            requestMemory: 2048
 
+    The above example command uses both ``--output`` and ``--output-run``.  The ``--output`` option creates the Chained collection if it doesn't already exist.  The ``--output-run`` option saves the unique Run collection that is also passed to all other compute jobs (i.e., one Run collection per submission).  If using both here, must include both ``--output`` and ``--output-run`` in the other ``runQuantumCommand``.
```

Comment:
Passing `--output` here will _redefine_ the CHAINED collection if it does exist, too, so maybe the second sentence should be:
```
The ``--output`` option creates the chained collection if necessary and defines it to include both the ``--input`` and `--output-run`` collections.
```

Also, this is a separate issue, so maybe a separate commit, but it would be good to remove `--skip-existing` here, as I'm pretty sure it doesn't do anything (it's only operative when making a QG _and_ using `--extend-run`), and it's possible `pipetask` might in the future make it an error to pass it when it doesn't make sense.

Reply:
First change done.  Testing without the --skip-existing in the --init-only command line.  If no problems during tests, I will make the change now before merging.

Reply:
The --skip-existing in the init-only job came from https://jira.lsstcorp.org/browse/DM-24797.  But doing a handful of tests, I cannot make the run have a problem without the --skip-existing.  Removing it.  Also removing --register-dataset-types from the init-only job based upon related slack discussions.

Reply:
More slack conversations about documentation for users to do instead of --register-dataset-types ended with leaving it in the command for now.